### PR TITLE
Add information about configuring cdn default ttl

### PR DESCRIPTION
**Pull request by tlwr**
[Story](https://www.pivotaltracker.com/n/projects/1275640/stories/167565279)

What
----

We updated the broker which is now different and more configurable

How to review
-------------

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).

Who can review
--------------

Not @tlwr

### CODE REVIEW

File: source/documentation/deploying_services/use_a_custom_domain.erb [hunk 1]
```diff
@@ -360,12 +360,27 @@ Custom domains are configured by our cdn-route service which uses a CloudFront d
 
 ### Caching
 
-CloudFront uses your app's `Cache-Control` or `Expires` HTTP headers to determine [how long to cache content](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html). If your app does not provide these headers, CloudFront will use a default timeout of 24 hours. This can be particularly confusing as different requests might be routed to different CloudFront Edge endpoints.
+CloudFront uses your app's `Cache-Control` or `Expires` HTTP headers to determine [how long to cache content](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html).
+
+If your app does not provide these headers, CloudFront will use the configured default time-to-live. This can be particularly confusing as different requests might be routed to different CloudFront Edge endpoints.
+
+The default time-to-live is set upon creation and is:
```
Comment (jonathanglassman) [inline source/documentation/deploying_services/use_a_custom_domain.erb]: What is created and what does the creating? When you create your CDN distribution?

If so, suggest: "When you create your CDN distribution, CloudFront sets the default time-to-live at:"

File: source/documentation/deploying_services/use_a_custom_domain.erb [hunk 2]
```diff
@@ -360,12 +360,27 @@ Custom domains are configured by our cdn-route service which uses a CloudFront d
...
+
+- 24 hours if your CDN distribution was created before 2019-08-12
```
Comment (jonathanglassman) [inline source/documentation/deploying_services/use_a_custom_domain.erb]: "- 24 hours if you created your CDN distribution before 2019-08-12"
More active
Same for second bullet

Comment (tlwr) [reply source/documentation/deploying_services/use_a_custom_domain.erb]: Resolved in 25a632bebe4593bd3b7b76ea5fc2ad98fb7d195a

File: source/documentation/deploying_services/use_a_custom_domain.erb (hunk 1 above)
Comment (tlwr) [reply source/documentation/deploying_services/use_a_custom_domain.erb]: Resolvedi in 25a632bebe4593bd3b7b76ea5fc2ad98fb7d195a

**Pull request merged**